core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -188,7 +188,7 @@ ${h.initPageVariables(context)}
                   </j:otherwise>
                 </j:choose>
                 <span style="white-space:nowrap">
-                  <a href="${rootURL}/user/${user.id}" class="model-link inside inverse"><b>${userName}</b></a>
+                  <a href="${rootURL}/user/me" class="model-link inside inverse"><b>${userName}</b></a>
```

Comment:
@fbelzunc What are we expecting to happen with this change? Does "me" resolve to something special on the server-side?